plaso/parsers/postgresql.py

```diff
@@ -0,0 +1,182 @@
+# -*- coding: utf-8 -*-
+"""Parser for PostgreSQL log files."""
+
+import string
+import pyparsing
+
+from dateutil import parser
+
+from dfdatetime import time_elements as dfdatetime_time_elements
+
+from plaso.containers import events
+from plaso.containers import time_events
+from plaso.lib import definitions, errors
+from plaso.parsers import logger
+from plaso.parsers import manager
+from plaso.parsers import text_parser
+
+class PostgreSQLEventData(events.EventData):
+  """PostgreSQL log data.
+
+  Attributes:
+    log_level (str): logging level of event
+    log_line (str): log message.
+    pid (int): process identifier (PID).
+    user (str): "user@database" string if present.
```

Comment:
please also explain what this value is, useraccount that was used to connect to the database?

Reply:
Done